shell/app-shell/elements/fb-data/fb-user.js

```diff
@@ -53,24 +53,34 @@ class FbUserElement extends Xen.Base {
     const path = `users/${userid}/arcs/${key}/touched`;
     log(`writing to [${path}]`);
     Firebase.db.child(path).set(Firebase.firebase.database.ServerValue.TIMESTAMP);
-   }
-  _queryUser({userid}, state) {
-    log('querying `user`');
-    if (state.field) {
-      state.field.dispose();
+  }
+  async _queryUser({userid}, state) {
+    if (await this._verifyUser(userid)) {
+      log('querying `user`');
+      if (state.field) {
+        state.field.dispose();
+      }
+      state.field = state.fbuser.queryUser(userid);
+      state.field.activate();
+    }
+  }
+  async _verifyUser(userid) {
+    const snap = await Firebase.db.child(`users/${userid}`).once('value');
+    if (snap.val() === null) {
+      this._fire('userid', '-L8ZV0oJ3btRhU9wj7Le');
```

Comment:
Seems worth adding a comment noting this hard coded id is intended to be the Barney user id as a fallback (and presumably that this is interim until we rework auth more fully).

Reply:
👍